src/stores/tests/wordpress-versions-api.test.ts

```
@@ -144,6 +144,9 @@ describe( 'WordPress Versions API', () => {
 	} );
 
 	it( 'should handle schema validation error for both channels', async () => {
+		const originalConsoleError = console.error;
+		console.error = jest.fn();
```

Maybe we should consider running Jest with the `--silent` option ([docs](https://jestjs.io/docs/cli#--silent)) instead? Seems like a good approach to me.

---

I looked into this but if I am not mistaken, I think this silences the results for all tests, not just the ones that are expected to produce errors. I think I would prefer to keep the logs working for tests that are not expected to produce errors so that we can be alerted if something gets messed with a specific setup. 

I went with `spyOn` to silence errors for specific tests where they are expected. We already use this approach in other tests as well.

---

Per the docs, it silences all console output from tests. Seems like it could save us some repeated boilerplate code in the tests. 

> We already use this approach in other tests as well.

Yeah, I know I've implemented this approach in the past, too. Defaulting to not printing console output seems like a better approach to me, though. We could discuss this separately if you prefer, though

---

> Yeah, I know I've implemented this approach in the past, too. Defaulting to not printing console output seems like a better approach to me, though. We could discuss this separately if you prefer, though

I will open a separate issue and we can discuss it there 👍

---

I added STU-493 to discuss this.